features/admin.applications.v1/components/settings/info.tsx

```diff
@@ -94,6 +100,13 @@ export const Info: FunctionComponent<InfoPropsInterface> = (
     const [ isWSFed, setIsWSFed ] = useState<boolean>(false);
     const [ isLoading, setIsLoading ] = useState<boolean>(false);
 
+    /**
+     * Index of the protocols tab.
+     */
+    const PROTOCOLS_TAB_INDEX: number = 1;
+
+    const isSAASDeployment: boolean = useSelector((state: AppState) => state?.config?.ui?.isSAASDeployment);
```

Comment:
Let's refrain from using `isSAASDeployment`. Let's use the extension model / deployment config instead

Reply:
fixed